### ISSUE TITLE
Hi, could u help me solve the following error, tks!

### ISSUE THREAD

**Issue by smartcai** [Vafali/Development-of-Deep-Reinforcement-Learning-policy-in-a-Parking-simulated-environment#4]
Select one option: 
        Play with trained model (press 0)
        Train pretrained model (press 1)
        Train new model (press 2)
        Your answer: 2
Enter the number of total episodes (for default [1000] press Enter):
        Your answer: 1000
Select model for training: 
        Only throttle action model (press 1)
        Throttle and steer actions model (press 2)
        Your answer: 2
Select spawning method: 
        Always in front of goal (press 1)
        Randomly somewhere around goal (press 2)
        Your answer: 2
Write the name of this training: 
        Your answer: model2312
Model: "Actor_Model"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
=================================================================
 input_1 (InputLayer)        [(None, 16)]              0

 batch_normalization (BatchN  (None, 16)               64        
 ormalization)

 dense (Dense)               (None, 512)               8704

 dense_1 (Dense)             (None, 256)               131328

 dense_2 (Dense)             (None, 2)                 514

=================================================================
Total params: 140,610
Trainable params: 140,578
Non-trainable params: 32
_________________________________________________________________
None
Model: "Critic_Model"
__________________________________________________________________________________________________
 Layer (type)                   Output Shape         Param #     Connected to
==================================================================================================
 input_2 (InputLayer)           [(None, 16)]         0           []

 input_3 (InputLayer)           [(None, 2)]          0           []

 batch_normalization_1 (BatchNo  (None, 16)          64          ['input_2[0][0]']
 rmalization)

 batch_normalization_2 (BatchNo  (None, 2)           8           ['input_3[0][0]']
 rmalization)

 dense_3 (Dense)                (None, 64)           1088        ['batch_normalization_1[0][0]']

 dense_5 (Dense)                (None, 64)           192         ['batch_normalization_2[0][0]']

 dense_4 (Dense)                (None, 128)          8320        ['dense_3[0][0]']

 dense_6 (Dense)                (None, 128)          8320        ['dense_5[0][0]']

 concatenate (Concatenate)      (None, 256)          0           ['dense_4[0][0]',
                                                                  'dense_6[0][0]']

 dense_7 (Dense)                (None, 256)          65792       ['concatenate[0][0]']

 dense_8 (Dense)                (None, 256)          65792       ['dense_7[0][0]']

 dense_9 (Dense)                (None, 1)            257         ['dense_8[0][0]']

==================================================================================================
Total params: 149,833
Trainable params: 149,797
Non-trainable params: 36
__________________________________________________________________________________________________
None
-----------------Start of training process---------------
Episode * 1 * Episodic Reward is ==> -960.076346052685 <<<>>> Agent crashed :(
Episode * 2 * Episodic Reward is ==> -993.8164936226331 <<<>>> Agent crashed :(
Episode * 3 * Episodic Reward is ==> -498.04477143016993 <<<>>> Agent moved far away from goal :(
_**[Errno 2] No such file or directory: 'D:\\01_caijia\\03_program\\ddpg\\CARLA_0.9.10\\PythonAPI\\ddpgcarla/recordings/throttle_and_steer/random_spawn/model2312/4.csv'**_
